index.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Title</title>
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/main.css">
+  </head>
+  <body>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#">
+          <img src="images/logo.svg" alt="logo">
+        </a>
+      </div>
+
+      <nav class="header__content">
+        <label for="delivery-time" class="header__delivery-time-lbl">
+          <input type="text" class="header__delivery-time" name="delivery-time" id="delivery-time" placeholder="ASAP">
+        </label>
+        <p class="header__to">to</p>
+        <label for="delivery-adress" class="header__delivery-adress-lbl">
+          <input type="text" class="header__delivery-adress" name="delivery-adress" id="delivery-adress" placeholder="Adress">
+        </label>
+        <a href="#" class="header__button header__sign-in-button">Sign In</a>
+        <a href="#" class="header__button header__register-button">Register</a>
+        <a href="#" class="header__basket">
+          <img src="images/basket.svg" alt="logo" class="header__basket-logo">
+        </a>
+      </nav>
+    </header>
+    <main>
+      <label for="search" class="main__search-lbl">
+        <input type="text" class="main__search" name="search" id="search" placeholder="Search for restaurant or cuisine">
```

Comment:
Переноси слишком длинные теги 
```suggestion
        <input
          type="text"
          class="main__search"
          name="search"
          id="search"
          placeholder="Search for restaurant or cuisine"
        >
```